mime-ext.yazi/init.lua

```diff
@@ -1026,26 +1026,56 @@ local EXTS = {
 	zsh = "text/shellscript",
 }
 
-local function fetch(self)
+local get_state = ya.sync(function (state)
+	return {
```

Comment:
Please just return `with_files` and `with_exts` here, instead of the merged result - merge them in the `peek` method in time to avoid unnecessary data transfer overhead between threads.

Reply:
done